test/BUILD

```diff
@@ -482,6 +482,18 @@ scala_binary(
     runtime_jdk = "@bazel_tools//tools/jdk:remote_jdk11",
 )
 
+# Make sure scala_library respects java_compile_toolchain during builds
+scala_junit_test(
+    name = "CheckBytecodeMajorVersion",
+    srcs = ["CheckBytecodeMajorVersionTest.scala"],
+    suffixes = ["Test"],
```

Comment:
for my learning, is `suffixes` significant?

Reply:
Yes, this is necessary for `scala_junit_test` to discover test classes. Specifically one of `prefixes` or `suffixes` is needed.

Reply:
there isn't a default so we must manually specify 😞